.drone.yml

```diff
@@ -174,6 +174,25 @@ steps:
     depends_on:
       - clone kube repo
 
+  - name: deploy to gamma
+    image: quay.io/ukhomeofficedigital/kd:v1.16.0
+    commands:
+      - cd kube-hocs-info-service
+      - ./deploy.sh
+    environment:
+      ENVIRONMENT: hocs-gamma
+      KUBE_TOKEN:
+        from_secret: hocs_info_service_hocs_gamma
+      VERSION: build_${DRONE_BUILD_NUMBER}
+      HOCS_INFO_SERVICE_DATA_VERSION: branch-epic_foi-mvp
```

Comment:
should this be `branch-epic/foi-mvp`?

Reply:
No; it's the Docker tag we're concerned with here, not the Git branch.